faust/app/base.py

```diff
@@ -984,13 +992,15 @@ def _on_rebalance_when_stopped(self) -> None:
                 await self.consumer.pause_partitions(assigned)
                 on_timeout.info('topics.on_partitions_assigned()')
                 await self.topics.on_partitions_assigned(assigned)
-                on_timeout.info('tables.on_partitions_assigned()')
-                await self.tables.on_partitions_assigned(assigned)
+                on_timeout.info('tables.on_rebalance()')
+                await self.tables.on_rebalance(
+                    assigned, revoked, newly_assigned)
                 on_timeout.info('+send signal: on_partitions_assigned')
                 await self.on_partitions_assigned.send(assigned)
                 on_timeout.info('-send signal: on_partitions_assigned')
                 on_timeout.info('flow_control.resume()')
                 self.flow_control.resume()
+                self._assignment = assigned
```

Comment:
Why are we setting this here again?

Reply:
Looks like it's left-over, it's already setting this in the line above. I removed this line.